packages/tangle/approvalweightmanager.go

```diff
@@ -127,7 +192,7 @@ func (a *ApprovalWeightManager) onSequenceSupportUpdated(marker *markers.Marker,
 		a.lastConfirmedMarkers[marker.SequenceID()] = currentMarker.Index()
 
 		fmt.Println("Marker confirmed!!")
-		a.Events.MarkerConfirmed.Trigger(currentMarker)
+		a.Events.MarkerConfirmed.Trigger(currentMarker, a.getMessageIDOfMarker(currentMarker, branchID))
```

Comment:
This is my bad, we should just trigger the event with the current marker and use the marker to message mapping as introduced in https://github.com/iotaledger/goshimmer/pull/1200 to get the corresponding message. Then we don't need `getMessageIDOfMarker` as well.